goosebit/settings/schema.py

```diff
@@ -61,9 +67,23 @@ def settings_customise_sources(
         dotenv_settings: PydanticBaseSettingsSource,
         file_secret_settings: PydanticBaseSettingsSource,
     ) -> tuple[PydanticBaseSettingsSource, ...]:
-        return (
+        settings_sources = [
             init_settings,
-            YamlConfigSettingsSource(settings_cls, BASE_DIR.joinpath("settings.yaml")),
-            env_settings,
-            file_secret_settings,
+            YamlConfigSettingsSource(settings_cls, GOOSEBIT_ROOT_DIR.joinpath("goosebit.yaml")),
+            YamlConfigSettingsSource(settings_cls, CURRENT_DIR.joinpath("goosebit.yaml")),
+        ]
+        if ETC_SETTINGS_FILE.exists():
+            settings_sources.append(
+                YamlConfigSettingsSource(settings_cls, ETC_SETTINGS_FILE),
+            )
+        if os.getenv("GOOSEBIT_SETTINGS") is not None:
+            settings_sources.append(
+                YamlConfigSettingsSource(settings_cls, os.getenv("GOOSEBIT_SETTINGS")),
+            )
+        settings_sources.extend(
+            [
+                env_settings,
+                file_secret_settings,
```

Comment:
We don't use secrets, so we can remove this, no?

Reply:
Sure.  The pydantic example I saw seems to want to include those always, but fine to remove.